index.bs

```diff
@@ -100,14 +100,14 @@ may be useful when doing so.
 <h2 id="questions">Questions to Consider</h2>
 
 <h3 class=question id="purpose">
-  What information might this feature expose to Web sites or other parties,
-  and for what purposes is that exposure necessary?
+  What information does this feature expose,
+  and for what purposes?
 </h3>
 
-User Agents should only expose information to the web
+User agents should only expose information to the Web
 when doing so is necessary to serve a clear user need.
-Does your feature expose information to origins?
-If so, how does exposing this information serve user needs?
+Does your feature expose information to wbe sites?
```

Comment:
*web

Reply:
Look, if you're going to expect me to spell everything correctly, we're going to be at this for a long time :P